docs/blueprints/EP036.rst

```diff
@@ -0,0 +1,237 @@
+
+Initially proposed here is a datastructured representing a linked list. 
+
+  .. code:: javascript
+
+    "pipeline": {
+      {
+        "id": "9cba7e27a33c45",
+        "status": "enabled",
+        "description": "First pipeline",
+        "multi_table": 
+        {
+          0: {
+            "table_miss_flow": {
+              "priority": 123, "match": {},
+              "instructions": [{"instruction_type": "goto_table", "table_id": 1}]
+            },
+            "napps_table_groups": {"coloring": ["base"],"of_lldp": ["base"]}
+          },
+          1: {
+            "table_miss_flow": {
+              "priority": 123, "match": {},
+              "instructions": [{"instruction_type": "goto_table","table_id": 2}]
+            },
+            "napps_table_groups": {"flow_manager": []}
+          },
+          2: {
+            "table_miss_flow": {
+              "priority": 123, "match": {},
+              "instructions": [{"instruction_type": "goto_table","table_id": 3}]
+            },
+            "napps_table_groups": {"mef_eline": ["evpl"]}
+          },
+          3: {
+            "table_miss_flow": {
+              "priority": 123, "match": {},
+              "instructions": [{"instruction_type": "goto_table","table_id": 5}]
+            },
+            "napps_table_groups": {"mef_eline": ["epl"]}
+          }
+          4: {
+            "table_miss_flow": {},
+            "napps_table_groups": {"telemetry_int": ["base"]}
+          }
+          5: {
+            "table_miss_flow": {},
+            "napps_table_groups": {}
+          }
+        }
+      }
+    }
+
+Events
+======
+
+Subscribed
+-----------
+
+  - **kytos/flow_manager.flow.added**
+  - **kytos/of_core.handshake.completed**
+  - **kytos/flow_manager.flow.error**
+  - **kytos/[mef_eline|telemetry_int|coloring|of_lldp].enable_table** - Every NApp sends an event with their table settings as content.
+  - **kytos/[mef_eline|telemetry_int|coloring|of_lldp].cookie** - Every NApp sends an event with their cookie information
+
+kytos/<napp_name>.enable_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event is sent from other NApps with information about their table settings.
+
+.. code:: javascript
+  "content": {
+    "mef_eline": {"epl": 3}
+  }
+
+kytos/<napp_name>.cookie
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: javascript
+  "content": {
+    "of_lldp": "0xac00000000000001"
+  }
+
+Published
+---------
+
+kytos/multi_table.enable_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event should sent before any NApp sends its first flow to be published.
+
+.. code:: javascript
+
+  "content": {
+    "mef_eline": {"epl": 3, "evpl": 2},
+    "of_lldp": {"base": 0},
+    "coloring": {"base": 0}
+  }
+
+Endpoint API
+============
+  
+  - GET `/api/kytos/multi_table/v1/pipeline/` - Get all pipelines in the database. Parameters enabled and disabled are allowed.

```

Comment:
@Alopalao could you please rename the routes too? since the Napp will be `of_multi_table` based on the last updates.